docs/_writing-tests/testdriver.md

```diff
@@ -18,6 +18,22 @@ the global scope.
 NB: presently, testdriver.js only works in the top-level test browsing
 context (and not therefore in any frame or window opened from it).
 
+### `test_driver.bless(intent, action)`
+#### `intent: a string describing the motivation for this invocation`
+#### `action: an optional function`
+
+This function simulates [activation][activation], allowing tests to
+perform privileged operations that require user interaction. For
+example, sandboxed iframes with the
+`allow-top-navigation-by-user-activation` may only navigate their
+parent's browsing context under these circumstances. The `intent`
```

Comment:
Having used this now, I still find myself having to look up what the right text is to include is for “intent”. What you be great is an example of an intent string, and how it ends up being concatenated (reviewing on my phone, so hard to look up what the final string is).

Reply:
An example sounds like a good idea to me! Added in a fixup commit